torch/utils/data/dataset.pyi

```diff
@@ -42,19 +44,33 @@ class IterableDataset(Dataset[T_co], metaclass=_DataPipeMeta):
     def set_reduce_ex_hook(cls, hook_fn: Any) -> None: ...
     @classmethod
     def set_getstate_hook(cls, hook_fn: Any) -> None: ...
+    # Functional form of 'BatcherIterDataPipe'
     def batch(self, batch_size: int, drop_last: bool = False, wrapper_class=DataChunk): ...
+    # Functional form of 'CollatorIterDataPipe'
     def collate(self, collate_fn: Callable= ..., fn_args: Optional[Tuple] = None, fn_kwargs: Optional[Dict] = None): ...
+    # Functional form of 'ConcaterIterDataPipe'
     def concat(self, *datapipes: IterDataPipe): ...
+    # Functional form of 'RoutedDecoderIterDataPipe'
     def decode(self, *handlers: Callable, key_fn: Callable= ...): ...
-    def demux(self, num_instances: int, classifier_fn: Callable[[T_co], int], drop_none: bool = False, buffer_size: int = 1000): ...
-    def filter(self, filter_fn: Callable, fn_args: Optional[Tuple] = None, fn_kwargs: Optional[Dict] = None, drop_empty_batches: bool = True, nesting_level: int = 0): ...
+    # Functional form of 'DemultiplexerIterDataPipe'
+    def demux(self, num_instances: int, classifier_fn: Callable[[T_co], Optional[int]], drop_none: bool = False, buffer_size: int = 1000): ...
+    # Functional form of 'FilterIterDataPipe'
+    def filter(self, filter_fn: Callable, fn_args: Optional[Tuple] = None, fn_kwargs: Optional[Dict] = None, drop_empty_batches: bool = True): ...
```

Comment:
Can you move the change of `map` and `filter` API to corresponding PRs?

Reply:
Done!